SMLHelper/Handlers/CraftDataHandlerBZ.cs

```diff
@@ -0,0 +1,319 @@
+﻿#if BELOWZERO
```

Comment:
Who came up with `BZ` and `Sub` suffixes?
That's a very poor way to organize the files in my opinion...